### ISSUE TITLE
Missing steps in the code

### ISSUE THREAD

**Issue by maiskovich** [HassanElmadany/Extract-SVO#2]
While running I get AttributeError: 'str' object has no attribute 'subtrees'

In the example the code just pass a string and then do `for s in t.subtrees(lambda t: t.label() == 'NP'):` without parsing or anything like that which results in the error described above

**Comment by HichemBouric** [comment]
Yes.. it's definetly missing.. you have to fill the gaps on your own
